source/_patterns/01-atoms/icons/logo.twig

```diff
@@ -1,7 +1,7 @@
  {% if img %}
   <div class="jcc-logo">
     {% include '@atoms/image/image.twig' with {
-      image: { src: img, alt: 'Alt text' }
+      image: { src: img.src, alt: img.alt }
```

Comment:
Quick fix to the structure that `image.twig` expects.  I got this working in Drupal too, thanks to your change.  🎉